test/box/gh-5998-one-tx-for-ddl.test.lua

```diff
@@ -0,0 +1,128 @@
+env = require('test_run')
+test_run = env.new()
+test_run:cmd("create server tx_man with script='box/tx_man.lua'")
+test_run:cmd("start server tx_man")
+test_run:cmd("switch tx_man")
+
+fiber = require('fiber')
+txn_proxy = require('txn_proxy')
+
+tx1 = txn_proxy.new()
+tx2 = txn_proxy.new()
+
+s1 = box.schema.space.create('s1')
+s1:create_index('pk')
```

Comment:
Please use `_ = s1:create_index.` constructions to avoid dependency on index output.

Reply:
Fixed.